### PR TITLE
Fix for negative feet/inch parsing

### PR DESCRIPTION
**Pull request by tmilnthorp**
Fix for #673.

### CODE REVIEW

**Comment by angularsen** [comment]
Looks good to me, only a failing test case.

**Comment by tmilnthorp** [comment]
> Looks good to me, only a failing test case.

Oops. I put "- 1′ 1″ " when I meant " -1′ 1″ ". Multi-line added the negative sign :) Fixed

That said, should "- 1′ 1″ "work?

**Comment by angularsen** [comment]
I don't know, but my intuition tells me we shouldn't have to support something like that. We have been bit previously by trying to be too lenient in parsing all sorts of input.

File: UnitsNet.Tests/CustomCode/LengthTests.FeetInches.cs [hunk 1]
```diff
@@ -41,17 +41,17 @@ public void FeetInchesRoundTrip()
...
-        [InlineData( "-1″", -0.08333333 )] // Inches only
-        [InlineData( "-1' 1\"", -1.08333333 )] // Normal form
-        [InlineData( "-1′ 1″", -1.08333333 )] // Normal form
-        [InlineData( "- 1′ 1″ ", -1.08333333 )] // Normal form, requires trimming
-        [InlineData( "-1'1\"", -1.08333333 )] // Without space
-        [InlineData( "-1′1″", -1.08333333 )] // Without space
-        [InlineData( "-1 ft 1 in", -1.08333333 )]
-        [InlineData( "-1ft 1in", -1.08333333 )]
+        [InlineData("-1'", 1)] // Feet only
+        [InlineData("-1′", 1)] // Feet only
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/LengthTests.FeetInches.cs]: Wait, what? Shouldn't `expectedFeet` still be negative 1?

Comment (tmilnthorp) [reply UnitsNet.Tests/CustomCode/LengthTests.FeetInches.cs]: Haha I re-copied-and-pasted. It's been a long day :) fixed

**Comment by angularsen** [comment]
Looks good now :-) green tests too!

**Comment by angularsen** [comment]
Would you publish a new release too, please?

**Comment by tmilnthorp** [comment]
Will do

**Pull request merged**